src/test/java/SeleniumMain.java

```diff
@@ -2,6 +2,7 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.edge.EdgeDriver;
+import utils.seleniumWrapper;
```

Comment:
**Issue Resolved**

Reply:
Please call the functions from selenium wrapper class like this :

line 37 :
SeleniumWrapper seleniumWrapper = new SeleniumWrapper();
seleniumWrapper.clickElement(By.id("selenium143"));